### DISCUSSION TITLE
What about print and PDF?

### DISCUSSION THREAD

**Discussion by sanskritkm** [Budibase/budibase#6965] [Ideas]
Hi every one,

This is just an idea what happen if we build a stock management and POS system and let cashier print out invoice or sent PDF by email or generate quotation for a customers. 🤪

**Comment by R2bEEaton** [comment]
Some of this can be achieved using the Embed component and (probably with a lot of effort) using an Automation with emails, but it would be amazing if there was native support for this.

Looks like it's TBD on the roadmap for this year.

https://github.com/Budibase/budibase/projects/10

(related #1372, #1353)
